content/account/settings.textile

```diff
@@ -0,0 +1,31 @@
+---
+title: Settings
```

Comment:
With the latest version, do you think this would make more sense just being part of the overview page, rather than standalone?